src/Apps/Feature/Components/FeatureFeaturedLink/FeatureFeaturedLink.tsx

```diff
@@ -0,0 +1,91 @@
+import React from "react"
+import styled from "styled-components"
+import { space, SpaceProps } from "styled-system"
+import { Box, Sans, Serif, Image, Flex } from "@artsy/palette"
+
+const Container = styled.a`
+  ${space}
+  display: flex;
+  text-decoration: none;
+  flex-direction: column;
+`
+
+const Figure = styled(Box)`
+  position: relative;
+`
+
+const Cover = styled(Image)`
+  display: block;
+  width: 100%;
+  height: 100%;
+`
+
+const Title = styled(Sans)`
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  background: linear-gradient(
+    rgba(255, 255, 255, 0) 0%,
+    rgba(0, 0, 0, 0.25) 100%
+  );
+`
+
+const Meta = styled(Flex).attrs({ mt: 1 })``
+
+const MetaCell = styled(Flex)`
+  flex-basis: 50%;
+`
+
+export type FeatureFeaturedLinkProps = SpaceProps & {
```

Comment:
#superminor but I don't think we _need_ the `Feature` namespacing in these component names since we're already nested under the `Feature` folder.

(This is not a strong opinion, this name in particular stuck out 😄)

Reply:
I'm of the opinion all subcomponents should be prefixed with their parent due to uniqueness constraints on relay fragment naming. This takes any thought out of it.

Reply:
Ah! That's... a good reason haha. 👍

Reply:
But yeah, I mean we don't do it currently AFAICT 🤷

Reply:
This is also the "Relay Way", as mentioned due to fragments -- it has to be such, otherwise we have the potential for name clashes. (I'm personally not so strict for no-relay components, but then things feel inconsistent.)